feat: Load more component

y-lohse opened the pull request:
This PR adds a new LoadMore component. [See the demo for more infos](https://yannick-lohse.fr/cozy-ui/react/#!/LoadMore)

y-lohse commented on react/LoadMore/index.jsx:
```diff
@@ -0,0 +1,51 @@
+    if (!isLoading) {
+      setIsLoading(true)
+      await fetchMore()
+      setIsLoading(false)
```
I considered adding a try/catch here… but we wouldn't be able to do anything with the error except maybe logging it. I think it's better to let it propagate.

ptbrowne commented on react/LoadMore/index.jsx:
```diff
@@ -0,0 +1,51 @@
+      await fetchMore()
+      setIsLoading(false)
+    }
+  }, [isLoading, fetchMore])
```
setIsLoading is also a dep right ?

y-lohse commented:
I've fixed the bug that prevented the button from being displayed, and I've changed the sizes of the blocks so that the spinner isn't visible on page load (which wasn't useful to understand how things work anyway). So the visual tests should be stable now!
